### PR TITLE
Additional Responsiveness to Properties page

### PR DESCRIPTION
**Pull request by kelfro**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: frontend/src/pages/Properties/Properties.tsx [hunk 1]
```diff
@@ -47,78 +48,80 @@ const Properties = () => {
...
-                        })}
-                      </select>
-                    </Cleaner>
-                  </CardBody>
-                </CardContent>
-              </HouseItem>
-            );
-          })
-        : null}
+			<PropContainer>
```
Comment (NandoTheessen) [inline frontend/src/pages/Properties/Properties.tsx]: I think either you aren't using prettier or it has gone rogue on you, this formatting is out of whack.

File: frontend/src/pages/Properties/PropertyDetails.tsx [hunk 2]
```diff
@@ -103,19 +105,23 @@ const PropertyDetails = (props: any) => {
...
-          <Top>
-            <MainText data-testid='house-detail'>{property.name}</MainText>
-            <SecondaryText>{property.address}</SecondaryText>
-            <BackButton text='Edit Property' colour='var(--colour-accent)' />
-            <BackButton
-              text='Go Back'
-              colour='var(--colour-accent)'
-              onClick={() => props.history.push('/properties')}
-            />
+					<Top>
```
Comment (NandoTheessen) [inline frontend/src/pages/Properties/PropertyDetails.tsx]: Same issue as above, can you fix this please?

File: frontend/src/pages/Properties/Properties.tsx (hunk 1 above)
Comment (kelfro) [reply frontend/src/pages/Properties/Properties.tsx]: Ok, I think it looks better now.

File: frontend/src/pages/Properties/PropertyDetails.tsx (hunk 2 above)
Comment (kelfro) [reply frontend/src/pages/Properties/PropertyDetails.tsx]: I think it looks better now!

File: frontend/src/pages/Properties/Properties.tsx [hunk 3]
```diff
@@ -48,80 +49,80 @@ const Properties = () => {
...
-              </HouseItem>
-            );
-          })
-        : null}
+      <PropContainer>
+        <HouseHeader>Recent Properties</HouseHeader>
+        <Link to='/properties/new'>
+          <Button text='New Property' />
+        </Link>
+        {loading ? '...Loading' : null}
```
Comment (NandoTheessen) [inline frontend/src/pages/Properties/Properties.tsx]: Sadly you overwrote the loadingIndicator that was added to master here, please re-add it.

Comment (kelfro) [reply frontend/src/pages/Properties/Properties.tsx]: Added!

**Comment by kelfro** [comment]
mobile

**Comment by kelfro** [comment]
details mobile

**Comment by kelfro** [comment]
details2 mobile

**Pull request merged**